NEWS.md

```diff
@@ -0,0 +1,243 @@
+    daily-reporting surveillance data source, as one might have assumed).
+
+## New features:
+
+* `as_epi_archive`, `epi_archive$new`:
+  * New `compactify` parameter allows removal of rows that are redundant for the
+    purposes of `epi_archive`'s methods, which use the last version of each
+    observation carried forward.
+  * New `clobberable_versions_start` field allows marking a range of versions
+    that could be "clobbered" (rewritten without assigning new version
+    tags); previously, this was hard-coded as `max(<epi_archive>$DT$version)`.
+  * New `versions_end` field allows marking a range of versions beyond
+    `max(<epi_archive>$DT$version)` that were observed, but contained no
+    changes.
+* `epix_merge`, `$merge`:
+  * New `sync` parameter controls what to do if `x` and `y` aren't equally
+    up to date (i.e., if `x$versions_end` and `y$versions_end` are
+    different).
+* New function `epix_fill_through_version`, method
+  `<epi_archive>$fill_through_version`: non-mutating & mutating way to
+  ensure that an archive contains versions at least through some
+  `fill_versions_end`, extrapolating according to `how` if necessary
+* Example archive data object is now constructed on demand from its
+  underlying data, so it will be based on the user's version of
+  `epi_archive` rather than an outdated R6 implementation from whenever the
+  data object was generated.
+
+# epiprocess 0.2.0:
+
+## Breaking changes:
+
+* Removed default `n=7` argument to `epix_slide`.
+
+## Improvements:
+
+* Ignore `NA`s when printing `time_value` range for an `epi_archive`.
+* Fixed misleading column naming in `epix_slide` example.
+* Trimmed down `epi_slide` examples.
+* Synced out-of-date docs.
+
+## Cleanup:
+
+* Removed dependency of some `epi_archive` tests on an example archive.
+  object, and made them more understandable by reading without running.
+* Fixed `epi_df` tests relying on S3 implementation external to `epiprocess`.
+* Added tests for `epi_archive` methods and wrapper functions.
+* Removed some dead code.
+* Made `.{Rbuild,git}ignore` files more comprehensive.
+
+# epiprocess 0.1.2:
+
+## New features:
+
+* New `new_epi_df` function is similar to `as_epi_df`, but (i) recalculates,
+  overwrites, and/or drops most metadata of `x` if it has any, (ii) may
+  still reorder the columns of `x` even if it's already an `epi_df`, and
+  (iii) treats `x` as optional, constructing an empty `epi_df` by default.
+
+## Improvements:
+
+* Fixed `geo_type` guessing on alphabetical strings with more than 2
+  characters to yield `"custom"`, not US `"nation"`.
+* Fixed `time_type` guessing to actually detect `Date`-class `time_value`s
+  regularly spaced 7 days apart as `"week"`-type as intended.
+* Improved printing of `epi_df`s, `epi_archives`s
+* Fixed `as_of` to not cut off any (forecast-like) data with `time_value >
+  max_version`.
+* Expanded `epi_df` docs include conversion from `tsibble`/`tbl_ts` objects,
```

Comment:
Inconsistent tense.  Should just decide.

Here and elsewhere: try to limit to a small number of verbs.  Maybe a recommended verb set, although some of these are trying to accomplish what the categories already do, but we might move from categories -> "importance", and then these would still be useful even though they might look redundant now.

Reply:
Partially resolved.  Rest spun out as part of another issue.